decidim-core/db/migrate/20180508111640_add_tos_version_to_organization.rb

```diff
@@ -8,8 +8,8 @@ class Organization < ApplicationRecord
   def up
     add_column :decidim_organizations, :tos_version, :datetime
     Organization.find_each do |organization|
-      tos_version = Decidim::StaticPage.find_by(slug: "terms-and-conditions", organization: organization).updated_at
-      organization.update(tos_version: tos_version)
+      tos_version = Decidim::StaticPage.find_by(slug: "terms-and-conditions", organization: organization)
+      organization.update(tos_version: tos_version.updated_at)
```

Comment:
@josepjaume I don't see what it solves this change, because if the terms-and-conditions page is missing the migration will also fail, maybe a `.try(:tos_version)` would not break the migration but the goal of the migration would not be achieved (set a tos_version to the organization).

Maybe something like this within the migration would be more helpful:

```rb
unless tos_version
  raise <<-MSG.squish
    Please create a terms-and-conditions page ...
  MSG
end
```

Reply:
You're definitely right, I messed up when replicating the changes I did in the local installation. Thanks for checking it out!

Breaking in the middle of a migration can be problematic - maybe we should create the missing pages there? Or maybe even create a separate migration to fix that.

Reply:
maybe creating a blank `terms-and-conditions` page, if missing,  within (or in a previous migration) is the best solution